docs/src/examples/Aerosol_activation_example.md

```diff
@@ -0,0 +1,51 @@
+# Aerosol Activation Example
+
+
+### Overview
+
+This example is based on AerosolActivation module which is a part of the
+  [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl) package.
+The AerosolActivation module computes the total number and mass
+  of aerosol particles that get activated and become cloud droplets,
+  given the atmospheric conditions and
+  the initial aerosol size distribution and properties.
+See the AerosolActivation module
+  [docs](https://clima.github.io/CloudMicrophysics.jl/dev/Aerosol_activation/)
+  for derivation and description of all input parameters.
```

Comment:
I think it is ok if you have docs for the aerosol activation code elsewhere, 

However could you write out the parameters and the data, more explicitly? Consider creating a "setting up the inverse problem" section to mimic that of `Cloudy` and `Lorenz`.

Reply:
will do